package.json

```diff
@@ -29,10 +30,10 @@
 		"enzyme-adapter-react-16": "^1.15.6",
 		"identity-obj-proxy": "^3.0.0",
 		"jest": "^27.0.4",
-		"node-sass": "^4.14.1",
+		"node-sass": "^6.0.0",
```

Comment:
Replace this with the `sass` package, I think it's a drop-in replacement (and then you won't need Python 2)